lib/Doctrine/Common/Collections/ExpressionBuilder.php

```diff
@@ -40,19 +40,19 @@ class ExpressionBuilder
      *
      * @return CompositeExpression
      */
-    public function andX($x = null)
+    public function andX(...$x)
```

Comment:
@stof strings not allowed [here](https://github.com/doctrine/collections/blob/master/lib/Doctrine/Common/Collections/Expr/CompositeExpression.php#L57)